build/pkgs/mathjax/spkg-configure.m4

```diff
@@ -0,0 +1,26 @@
+SAGE_SPKG_CONFIGURE([mathjax], [
+  # Arch:   /usr/share/mathjax
+  # Gentoo: /usr/share/mathjax
+  # Void:   /usr/share/mathjax
+  AC_MSG_CHECKING([for MathJax-3.x])
+  m4_foreach([mathjax_dir], [/usr/share/mathjax], [
+    # tex-chtml.hs is used in src/sage_docbuild/conf.py
+    # and was not present in MathJax-2.x
+    AS_IF([test -f "mathjax_dir/tex-chtml.js"], [
+      SAGE_MATHJAX_DIR="mathjax_dir"
+      AC_MSG_RESULT([mathjax_dir])
```

Comment:
There seems no easy fix. A trick is to add ".." to `mathjax_dir` here so that we use ".." to indicate that this is a system one, and then use the indicator to differentiate how to tell sphinx where to find mathjax in `conf.py`.